app/src/components/governance/PageProposals.vue

```diff
@@ -23,6 +23,7 @@
       />
     </div>
     <ModalPropose
+      v-if="parameters && Object.keys(parameters).length > 0"
```

Comment:
what's this about?

Reply:
there are no governance parameters in Polkadot set, as it is always very flexible and varying according to the proposal

Reply:
sure but what does this have to do with hiding / showing the proposal modal?

Reply:
the component is there, being called without v-if, which queries for stuff that Polkadot hasn't got.

We will really fix this once we implement the voting action, but for now this is better

Reply:
can you leave a comment about future refactor then?

Reply:
Done